library/java/org/chromium/net/impl/NativeCronetEngineBuilderImpl.java

```diff
@@ -61,7 +61,11 @@ public class NativeCronetEngineBuilderImpl extends CronetEngineBuilderImpl {
   private String mAppId = "unspecified";
   private TrustChainVerification mTrustChainVerification = VERIFY_TRUST_CHAIN;
   private String mVirtualClusters = "[]";
-
+  private List<EnvoyHTTPFilterFactory> platformFilterChain = Collections.emptyList();
+  private List<EnvoyNativeFilterConfig> nativeFilterChain = new ArrayList<>();
```

Comment:
I figured, for consistency sake, the variables should either be in/out of the methods since they aren't used by other methods.

I'll add a refactor todo to move the variable into the methods.